.github/workflows/run-tests-build-docs.yml

```diff
@@ -0,0 +1,71 @@
+name: run-tests-build-docs
```

Comment:
Should we maybe rename the ERT/Everest workflows a bit to clarify which is Everest and which is ERT?

Reply:
I second the above. Makes it easier in the "checks" part the PRs to see which part failed.

Reply:
The name of the workflow files should also be changed to make it clear if they are from ERT or Everest

Reply:
Renamed them now